theory/energetic sources/AGNs.md

```diff
@@ -7,3 +7,35 @@ nav_order: 1
 ---
 
 ## Active Galactic Nuclei (AGNs)
+
+### Introduction
+
+Active Galactic Nuclei or AGNs as they are known, are very powerful and bright galactic centers, powered by supermassive [black holes](../special%20stars/black%20holes.html). You can think of these black holes as cosmic vacuum cleaners of enormous size, pulling the surrounding gas and dust toward themselves.
+
+This material, as it approaches the black hole, begins to heat up due to friction and then radiates light, emitting energy across the whole spectrum-from radio waves to X-rays. This is the reason why AGNs are so bright; sometimes they outshine all the stars in their host galaxy put together. They are mostly situated at the center of the host galaxy.
+
+### Types of AGNs
+
+Varieties of [AGNs](#active-galactic-nuclei-agns) exist, and their appearance also depends on how we're looking at them. For example, quasars are AGNs that are extremely bright and can be seen from billions of light-years away. Not all AGNs are [quasars](../energetic%20sources/quasars.html). Some [quasars](../energetic%20sources/quasars.html) are so far away that the light we see today started its journey when the universe was still young.
+
+Seyfert galaxies are a kind of AGN, residing in spiral galaxies much like the Milky Way. But they are less powerful compared to quasars. [Blazars](../energetic%20sources/blazars.html) are [AGNs](#active-galactic-nuclei-agns) in which one of the two high-energy jets emanating from the [black hole](../special%20stars/black%20holes.html) happens to point directly toward Earth, which makes them exceptionally bright.
+
+### Structure of AGNs
+
+[AGNs](#active-galactic-nuclei-agns) generally have a supermassive [black hole](../special%20stars/black%20holes.html), at the core which usually hosts a mass ranging from millions to billions of times the mass of the Sun. Around the SMBH is an accretion disk, comprising gas and dust spiraling inward with the immense gravitational pull of the [black hole](../special%20stars/black%20holes.html).
```

Comment:
Please put "(SMBH)" after the mention of a super-massive black hole for the first time and then proceed to use the acronym

Reply:
Resolved now.